RELEASE_NOTES.md

```diff
@@ -1,5 +1,38 @@
 # Release Notes
 
+## 2.1.6 "Rama" - June 4, 2020
+
+* There are a number of new default overrides to increase Agent throughput:
+
+  ```
+  "max_allowed_request_size": 5900000
+  "pipeline_threshold": 0
+  "min_request_spacing_interval": 0.0
+  "max_request_spacing_interval": 5.0
+  "max_log_offset_size": 200000000
+  "max_existing_log_offset_size": 200000000
+  ```
+
+  Increased throughput may result in a larger amount of logs uploaded to Scalyr if the Agent has been skipping logs
+  before this upgrade, and as a result a larger bill.
+
+  If you are interested in avoiding higher throughput, these options are tied to the new `max_send_rate_enforcement`,
+  you can disable these overrides by setting `max_send_rate_enforcement` to `"legacy"`.
+  If you want to set a rate value for `max_send_rate_enforcement` but still disable the overrides you need to set
```

Comment:
Remove this sentence.  `disable_max_send_rate_enforcement_overrides` is meant to be a hidden option.